bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```diff
@@ -246,11 +246,11 @@ const ChatMessageList: React.FC<ChatListProps> = ({
               ) : null
           }
         </span>
-        <div id="contentRef" ref={contentRef}>
+        <div id='contentRef' ref={contentRef}>
           <ChatPopupContainer />
           {
             // @ts-ignore
-            Array.from({ length: pagesToLoad }, (v, k) => k + (firstPageToLoad)).map((page) => {
+            Array.from({ length: pagesToLoad }, (_v, k) => k + firstPageToLoad).map((page) => {
```

Comment:
why did you add the underscore to the v parameter? it's not used, only declared because we need access k value